lib/src/main/java/org/team100/lib/localization/NotePosition24ArrayListener.java

```diff
@@ -43,8 +48,22 @@ void consumeValues(NetworkTableEvent e) {
             } catch (RuntimeException ex) {
                 return;
             }
-
+            notes = new Optional[] { Optional.empty() };
+            int noteNum = 0;
+            Optional<Double> y = Optional.empty();
```

Comment:
why are these declarations here, when the values are only ever used inside the loop?